### DISCUSSION TITLE
Question: what does this button do?

### DISCUSSION THREAD

**Discussion by Altonss** [CatimaLoyalty/Android#632] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/66519591/143879798-49e20a18-4b4b-4d49-a02b-406384568be4.png)
What does this button do? When I press it I don't notice any change...
![image](https://user-images.githubusercontent.com/66519591/143879840-2e2a446c-9303-4f0c-befe-9dd9ce84af76.png)

**Comment by TheLastProject** [comment, marked as answer]
The icon toggles the "expanded" or "collapsed" display. Collapsed display allows more cards to fit on the screen by hiding some "less important" fields.

✔️ = Visible
❌ = Hidden

|                    | Expanded | Collapsed |
|-----------------|---------------|---------------|
| **Name**    | ✔️    | ✔️   |
| **Note**      | ✔️    | ❌   |
| **Balance** | ✔️    | ❌   |
| **Expiry**    | ✔️    | ❌   |

**Reply by Altonss** [reply]
Ah I understand why it changes nothing for me, as I don't have Note, Balance or Expiry on any of my cards. Thanks :)

**Reply by TheLastProject** [reply]
As of Catima 2.13.0, the button now also makes the card header (icon/image) smaller, so that it is useful even if you don't have any notes, balance or expiry on any of your cards.
